e2e/libs/longhorn_deploy/longhorn_deploy.py

```diff
@@ -47,19 +48,20 @@ def install(self, install_stable_version):
         installed = self.longhorn.install(install_stable_version)
         if not installed:
             logging(f"Installing Longhorn failed")
-            time.sleep(self.retry_count)
+            time.sleep(self.retry_interval)
```

Comment:
If it only sleeps for `self.retry_interval` second, it only sleeps for `1` second before failing the test case and tearing down the test environment, leaving us no opportunity to investigate the issue.